a10_octavia/controller/worker/flows/a10_member_flows.py

```diff
@@ -129,31 +129,79 @@ def get_delete_member_flow(self):
         delete_member_flow.add(vthunder_tasks.SetupDeviceNetworkMap(
             requires=a10constants.VTHUNDER,
             provides=a10constants.VTHUNDER))
-        delete_member_flow.add(server_tasks.MemberDelete(
-            requires=(constants.MEMBER, a10constants.VTHUNDER, constants.POOL)))
         delete_member_flow.add(database_tasks.DecrementMemberQuota(
             requires=constants.MEMBER))
-        if CONF.a10_global.network_type == 'vlan':
-            delete_member_flow.add(vthunder_tasks.DeleteInterfaceTagIfNotInUseForMember(
-                requires=[constants.MEMBER,
-                          a10constants.VTHUNDER]))
+        delete_member_flow.add(self.get_delete_member_vthunder_subflow())
         delete_member_flow.add(database_tasks.MarkPoolActiveInDB(
             requires=constants.POOL))
         delete_member_flow.add(database_tasks.
                                MarkLBAndListenersActiveInDB(
                                    requires=[constants.LOADBALANCER,
                                              constants.LISTENERS]))
-        delete_member_flow.add(a10_database_tasks.GetVRIDForProjectMember(
+        delete_member_flow.add(self.get_delete_member_vrid_subflow())
+        delete_member_flow.add(vthunder_tasks.WriteMemory(
+            requires=a10constants.VTHUNDER))
+        return delete_member_flow
+
+    def get_delete_member_vthunder_subflow(self):
+        delete_member_vthunder_subflow = linear_flow.Flow(
+            a10constants.DELETE_MEMBER_VTHUNDER_SUBFLOW)
```

Comment:
`get_delete_member_vthunder_subflow` and `get_delete_member_vthunder_internal_subflow` shouldn't have the same subflow name